supplementary_style_guide/glossary_terms_conventions/general_conventions/x.adoc

```diff
@@ -9,6 +9,43 @@
 
 *See also*: xref:xemacs[XEmacs]
 
+[discrete]
+[[x86_64]]
+==== image:images/yes.png[yes] x86_64 (noun)
+*Description*: A 64-bit version of x86 architecture. Use this format when referring to an Operating systems and server instances. Including: Linux versions, RHEL, or CoreOS. Use in backticks when referring to architecture as a value or parameter.
```

Comment:
```suggestion
*Description*: A 64-bit version of x86 architecture. Use this format when referring to Operating systems and server instances. Including: Linux versions, RHEL, or CoreOS. Use in backticks when referring to architecture as a value or parameter.
```

Reply:
```suggestion
*Description*: A 64-bit version of the x86 architecture. Use this format when referring to operating systems and server instances. Including: Linux versions, RHEL, or CoreOS. Use backticks when referring to the architecture as a value or a parameter.
```

Reply:
Yeah I agree and think that looks better